addons/account/models/ir_http.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+from odoo import models
+
+
+class IrHttp(models.AbstractModel):
+    _inherit = 'ir.http'
+
+    def session_info(self):
+        result = super(IrHttp, self).session_info()
+        if self.env.user._is_internal() and self.env.company.quick_edit_mode:
+            result['is_quick_edit_mode_enabled'] = True
```

Comment:
```suggestion
        result['is_quick_edit_mode_enabled'] = self.env.user._is_internal() and self.env.company.quick_edit_mode
```
We should do it like this to make sure the key is always set.